libs/libm/newlib/Make.defs

```diff
@@ -88,6 +80,13 @@ CSRCS += $(wildcard newlib/newlib/newlib/libm/machine/$(ARCH)/*.c)
 VPATH += :newlib/newlib/newlib/libm/machine/$(ARCH)
 DEPPATH += --dep-path newlib/newlib/newlib/libm/machine/$(ARCH)
 
+CSRCS += $(wildcard newlib/newlib/newlib/libm/common/*.c)
```

Comment:
what's about cmake

Reply:
Let me check it...

Reply:
Hi @xiaoxiang781216 . I've run some local tests and the CMake-based build does not suffer of the same problem (I've found another bug related to including the toolchain's libm, but it isn't related to this PR and I'll submit another PR to fix it). However, to maintain consistency, I changed the order of inclusion for CMake, too!